main.go

```diff
@@ -224,6 +225,11 @@ func main() {
 	// Get first container definition to create slack message
 	containerDef = taskDesc.TaskDefinition.ContainerDefinitions[0]
 
+	var appDisplayVersion string
+	if *appVersion != "" {
+		appDisplayVersion = fmt.Sprintf(" (version %s) ", *appVersion)
```

Comment:
```suggestion
		appDisplayVersion = fmt.Sprintf(" (version %s)", *appVersion)
```
Removing trailing space here so that the message would looke like
`*webregister (version 1234)*` rather than `*webregister (version 1234) *`
though not sure if even matters for Slack